csss-site/src/events/frosh/templates/frosh/2023/frosh.html

```diff
@@ -0,0 +1,257 @@
+				by the CSSS at SFU.
+			</p>
+			<div class="buttons">
+				<button onclick="location.href='https://forms.gle/fLwTi15iZy8qnySPA'" class="rounded leader">
+					<img src="{% static 'frosh_static/2023/images/register.svg' %}" alt="Register">
+					Sign-Up
+				</button>
+				<button onclick="location.href='#faq'" class="rounded">
+					Read FAQ
+				</button>
+			</div>
+			<div class="spacer"></div>
+			<h2>Who is "the CSSS"? <span class="small">(???)</span></h2>
+			<p>
+				The <b>Computing Science Student Society</b> (CSSS)
+				is a student-led organization at Simon Fraser
+				University that represents undergraduate students
+				studying Computing Science.
+			</p>
+			<p>
+				Students that are declared Computing Science majors,
+				minors, or are taking CS courses, are automatically
+				members of the society.
+			</p>
+			<p>
+				To connect with the CSSS, check out our <b>Discord
+				server</b>, or visit our two common rooms located in
+				the Applied Science Building at SFU's Burnaby
+				campus: <b>ASB 9802</b>, and <b>ASB 9971</b>.
+			</p>
+			<div class="buttons">
+				<button onclick="location.href='https://discord.gg/sfucsss'" class="rounded">
+					<img src="{% static 'frosh_static/2023/images/discord.svg' %}" alt="Discord">
+					CSSS Discord Server
+				</button>
+				<button onclick="location.href='https://sfucsss.org'" class="rounded">
+					<img src="{% static 'frosh_static/2023/images/CSSS_white.svg' %}" alt="CSSS">
+					CSSS Website
+				</button>
+			</div>
+			<div class="spacer"></div>
+			<h2>What is <b>RETRO FROSH</b>?</h2>
+			<p>
+				<b>RETRO FROSH</b> is the name given to the
+				Frosh Week of 2023.
+			</p>
+			<p>
+				Every Frosh Week has a theme: for example, last
+				year's was Candy Frosh, the year before that...
+				something to do with Zelda, I think.
+			</p>
+			<p>
+				This year is SFU's School of Computing Science's
+				50<sup>th</sup> anniversary. To acknowledge this,
+				we decided to theme 2023's Frosh Week as <b>RETRO
+				FROSH</b>.
+			</p>
+			<div class="spacer"></div>
+			<h2>When?</h2>
+			<p>
+				RETRO FROSH starts <b>Monday, September 11<sup>th</sup></b>
+				and continues until <b>Saturday, September 16<sup>th</sup></b>.
+				Information about the time and location of each event
+				can be found in the <a href="#schedule">schedule</a>.
+			</p>
+			<div class="spacer"></div>
+			<h2>How can I join?</h2>
+			<p>
+				Please <b>Sign-Up</b> to secure your place in
+				RETRO FROSH. Attendees receive a free RETRO
+				FROSH themed T-Shirt, as well as several
+				free meals throughout the week.
+			</p>
+			<div class="spacer"></div>
+			<h2>Gallery</h2>
+			<div class="imgs">
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery1.png' %}" alt="Frosh Week 2021">
+					<p>Frosh Week 2021: Hike</p>
+				</div>
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery2.png' %}" alt="Gallery 1">
```

Comment:
fix inconsistent alt text naming

Reply:
Thanks Gabe, new commits reflect changes to fix the issues you mentioned.